go/simple/v2/examples/simple_example/simple_example.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"ampli-example/ampli"
+	"github.com/amplitude/analytics-go/amplitude"
+	"github.com/joho/godotenv"
+	"log"
+	"os"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatalln("Error loading .env file")
+	}
+	apiKey := os.Getenv("AMPLITUDE_API_KEY")
+
+	// Initialize the Ampli instance with LoadOptions and LoadClientOptions
+	ampli.Instance.Load(ampli.LoadOptions{
+		Client: ampli.LoadClientOptions{
+			Configuration: amplitude.NewConfig(apiKey),
+		},
+	})
+	defer ampli.Instance.Shutdown()
+
+	// Identify using IdentifyProperties in tracking plan
+	identifyObj := ampli.NewIdentify(16.6).SetOptionalArray([]string{"abc", "test"})
+	ampli.Instance.Identify("user_id", identifyObj, amplitude.EventOptions{})
```

Comment:
Maybe we should use `optional argument` pattern to not pass empty options explicitly:
```
func (a *Ampli) Identify(userID string, identify *Identify, eventOptions ...amplitude.EventOptions) {
	a.Track(userID, identify, eventOptions...)
}
```

```
ampli.Instance.Identify("user_id", identifyObj)
```

Reply:
But in that case, EventOptions would be an array. What if users pass multiple `EventOptions`? Do we only simply accept the first one?

Reply:
Yes, the first one.

Reply:
changed to it optional as suggested above.

Reply:
1. Should be 
```
func (a *Ampli) EventNoProperties(userID string, event *EventNoProperties, eventOptions ...EventOptions) {
	a.Track(userID, event, eventOptions...)
}
```
instead of
```
func (a *Ampli) EventNoProperties(userID string, event *EventNoProperties, eventOptions ...EventOptions) {
	a.Track(userID, event, eventOptions[0])
}
```
2. `eventOptions[0]` fails if `eventOptions` is empty